src/lavinmq/launcher.cr

```diff
@@ -202,9 +206,19 @@ module LavinMQ
       puts "  marker threads: #{ps.markers_m1}"
       puts "  reclaimed bytes during last GC: #{ps.bytes_reclaimed_since_gc.humanize_bytes}"
       puts "  reclaimed bytes before last GC: #{ps.reclaimed_bytes_before_gc.humanize_bytes}"
-      puts "Fibers:"
-      Fiber.list { |f| puts f.inspect }
-      LavinMQ::Reporter.report(@amqp_server)
+      # puts "Fibers:"
+      # Fiber.list { |f| puts f.inspect }
+      # LavinMQ::Reporter.report(@amqp_server)
+      {% unless flag?(:release) %}
+        puts "Object counts:"
+        PerfTools::MemProf.log_object_counts(STDOUT)
+        puts "Object sizes:"
+        PerfTools::MemProf.log_object_sizes(STDOUT)
+        puts "Allocations:"
+        PerfTools::MemProf.log_allocations(STDOUT)
+        puts "Fibers:"
+        PerfTools::FiberTrace.log_fibers(STDOUT)
```

Comment:
Output from `log_fibers` 

```
4
/usr/lib/crystal/crystal/system/unix/signal.cr:99:7 in 'process'
/usr/lib/crystal/crystal/system/unix/signal.cr:87:9 in '->'
/usr/lib/crystal/fiber.cr:148:11 in 'run'
/usr/lib/crystal/fiber.cr:100:34 in '->'

....
```

what does it mean and is it worth to have in the output?